client/my-sites/post-type-list/index.jsx

```diff
@@ -153,6 +154,7 @@ class PostTypeList extends Component {
 						siteId={ siteId }
 						query={ { ...query, page } } />
 				) ) }
+				{ siteId && <QueryPostTypes siteId={ siteId } /> }
```

Comment:
>Why is that so and how is it bad?

Because requests are triggered by the mount. I assume `<PopoverMenu />` is only rendering the menu items when the menu is visible. I'd thought to move it to the base menu component, but this is even worse in that requests are triggered both after the initial request of posts and when VirtualScroll changes the rendered indices.

Whether it's "bad" that types are requested on each visible toggle... maybe not. Could potentially be a bad user experience if they've not yet been loaded (but it's likely they have been, since My SItes sidebar requires the types). Likely quite a bit of unnecessary requesting, though.

Almost wonder if we should drop the post type specific labels, since they're already a bit inconsistent (there's no "Trash Project" or "Publish Project" labels available). "View", "Edit" could serve just as well across any post type? The Edit component also pulls capability from the post type, but I'm a bit worried about this anyways, since our site endpoint doesn't return _all_ of the capabilities (nothing to stop someone from setting the edit capability for their custom post type to `switch_themes`).